Tests/ttLib/tables/ttProgram_test.py

```diff
@@ -101,19 +100,19 @@ def test_roundtrip(self):
         assert BYTECODE == p.getBytecode()
 
     def test_xml_indentation(self):
-        with open(TTPROGRAM_TTX, 'rb') as f:
-            ttProgramXML = f.read().splitlines()
+        with open(TTPROGRAM_TTX, 'r') as f:
```

Comment:
it's good practice to always use `encoding='utf-8'` when reading text files which we know are encoded as UTF-8, since on Windows `sys.getdefaultencoding()` does not returns 'UTF-8', but some narrow codepage. Though in this case it should be fine since the `ttProgram.ttx` only contains ascii.